src/command_modules/azure-cli-sqlvirtualmachine/HISTORY.rst

```diff
@@ -0,0 +1,9 @@
+.. :changelog:
+
+Release History
+===============
+
+0.0.1 (2017-03-01)
+++++++++++++++++++++++++++
```

Comment:
This should be version 0.1.0 and please remove the date. It looks like it is supposed to be the release date, in which case it is wildly inaccurate. If it's not the release date (maybe an API version for example) then it's misleading.

Reply:
Done :)